documentation/ame-guide/modules/ROOT/pages/migration-of-models.adoc

```diff
@@ -0,0 +1,54 @@
+:page-partial:
+
+[[migration-of-models]]
+= Migration of models
+
+There are three different ways to migrate Aspect Model:
+
+* Automatic migration when starting the application.
+* Delayed migration in the sidebar.
```

Comment:
* Automatic migration when starting the application.
* Initiate migration from the sidebar.
* Loading a (syntactically valid) Aspect Model based on an olders BAMM version